pandas/tests/indexes/datetimes/test_setops.py

```diff
@@ -367,6 +367,19 @@ def test_join_nonunique(self):
         rs = idx1.join(idx2, how="outer")
         assert rs.is_monotonic
 
+    @pytest.mark.parametrize("tz", [None, "US/Pacific"])
```

Comment:
Could we use the `tz_aware_fixture` here?

Reply:
sure